src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -156,7 +156,7 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdUti
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
         $$rAF(function animation(now) {
-          var currentTime = (now || getTimestamp()) - startTime;
+          var currentTime = $window.Math.min((now || getTimestamp()) - startTime, animationDuration);
```

Comment:
Just asked, because to be consistent, we always use it without `$window`. [See here](https://github.com/angular/material/blob/master/src/components/input/input.js#L433)

But it's your choice :wink: